server/routes/router.js

```diff
@@ -15,4 +17,10 @@ router.get('/api/v1/users', UsersController.getUsers)
   .put('/api/v1/users/:id', UsersController.updateUser)
   .delete('/api/v1/users/:id', UsersController.deleteUser);
 
+// Documents endpoint
+router.get('/api/v1/documents', DocsController.getDocuments)
```

Comment:
You might want to take a look a `router.route` if you haven't already. Just for future use though.